pptx_blueprint/__init__.py

```diff
@@ -37,7 +37,27 @@ def replace_picture(self, label: str, filename: _Pathlike) -> None:
             label (str): label of the placeholder (without curly braces)
             filename (path-like): path to an image file
         """
-        pass
+        shapes_to_replace = self._find_shapes(label)
+        if not shapes_to_replace:
+            return
+        if isinstance(filename, str):
+            filename = pathlib.Path(filename)
+        if not filename.is_file():
+            raise FileNotFoundError(f"The file does not exist: {filename}")
+        img_file = open(filename, "rb")
+        old_shape: BaseShape
```

Comment:
I would maybe type `shapes_to_replace`, instead of old_shape.

Reply:
I already use `shapes_to_replace` for the iterator that is returned by `_find_shapes()`. In my opinion the singular version `shape_to_replace` is to close to `shapes_to_replace` and therefor `old_shape` makes it easier to distinguish between the iterator and the single shape.